AndroidStealth/src/main/AndroidManifest.xml

```diff
@@ -1,102 +1,112 @@
+				android:exported="false"
+				>
 			<intent-filter>
-				<action android:name="encryption.EncryptionService.TAP_TO_LOCK" />
+				<action android:name="encryption.EncryptionService.TAP_TO_LOCK"/>
 			</intent-filter>
 		</service>
 
 		<activity
-			android:name="com.ipaulpro.afilechooser.FileChooserActivity"
-			android:enabled="@bool/use_activity"
-			android:exported="true"
-			android:icon="@drawable/ic_chooser"
-			android:label="@string/choose_file" >
+				android:name="com.ipaulpro.afilechooser.FileChooserActivity"
+				android:enabled="@bool/use_activity"
+				android:exported="true"
+				android:icon="@drawable/ic_chooser"
+				android:label="@string/choose_file"
+				>
 			<intent-filter>
-				<action android:name="android.intent.action.GET_CONTENT" />
+				<action android:name="android.intent.action.GET_CONTENT"/>
 
-				<category android:name="android.intent.category.DEFAULT" />
-				<category android:name="android.intent.category.OPENABLE" />
+				<category android:name="android.intent.category.DEFAULT"/>
+				<category android:name="android.intent.category.OPENABLE"/>
 
-				<data android:mimeType="*/*" />
+				<data android:mimeType="*/*"/>
 			</intent-filter>
 		</activity>
 		<activity
-			android:name="com.stealth.pin.PinActivity"
-			android:excludeFromRecents="true"
-			android:theme="@style/AppTheme.Dialog" >
+				android:name="com.stealth.pin.PinActivity"
+				android:excludeFromRecents="true"
+				android:theme="@style/AppTheme.Dialog"
+				>
 			<intent-filter>
-				<action android:name="android.intent.action.MAIN" />
+				<action android:name="android.intent.action.MAIN"/>
 
-				<category android:name="android.intent.category.LAUNCHER" />
+				<category android:name="android.intent.category.LAUNCHER"/>
 			</intent-filter>
 		</activity>
 
-		<activity android:name="net.rdrei.android.dirchooser.DirectoryChooserActivity" />
+		<activity android:name="net.rdrei.android.dirchooser.DirectoryChooserActivity"/>
 
-		<receiver android:name="com.stealth.android.StealthButton" >
+		<receiver android:name="com.stealth.android.StealthButton">
 			<intent-filter>
-				<action android:name="android.appwidget.action.APPWIDGET_UPDATE" />
-				<action android:name="com.stealth.android.StealthButton.ACTION_BUTTON_PRESS" />
+				<action android:name="android.appwidget.action.APPWIDGET_UPDATE"/>
+				<action android:name="com.stealth.android.StealthButton.ACTION_BUTTON_PRESS"/>
 			</intent-filter>
 
 			<meta-data
-				android:name="android.appwidget.provider"
-				android:resource="@xml/stealth_button_info" />
+					android:name="android.appwidget.provider"
+					android:resource="@xml/stealth_button_info"
+					/>
 		</receiver>
 
 		<activity
-			android:name="com.stealth.android.RecorderActivity"
-			android:label="@string/title_activity_recorder" >
-		</activity>
+				android:name="com.stealth.android.RecorderActivity"
+				android:label="@string/title_activity_recorder"
+				></activity>

```

Comment:
The problem is that Android Studio uses tab-width 4, and github uses tab-width 8. In Android Studio, this looks ok.